.github/workflows/CMake-minimal-version.yaml

```diff
@@ -0,0 +1,41 @@
+name: CMake minimal version test
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test_downloader:
+    name: "Test cmake_downloader.py"
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: "Run test"
+        run: |
+          sudo apt-get install  xorg-dev libglu1-mesa-dev
+          
+          cd ..
+          git clone https://github.com/nlohmann/cmake_min_version.git --depth 1
+          cd cmake_min_version
+          
+          python3 -mvenv venv
+          venv/bin/pip3 install -r requirements.txt
+          venv/bin/python3 cmake_downloader.py --latest_release
+
+          output=$(venv/bin/python3 cmake_min_version.py /home/runner/work/librealsense/librealsense/)
+          cmake_version=$($output | grep "cmake_minimum_required(VERSION"))
+          
+          echo $cmake_version
+          
+          if [ $cmake_version -ne "cmake_minimum_required(VERSION 3.8.0)" ]
+          then
+            echo "$cmake_version found"
+            echo "Error: A minimal CMake version is not correct!"
```

Comment:
`Error - LibRS minimal CMake version require is <x> but on this build the minimal is <y>`

Reply:
Done